python/app.py

```diff
@@ -12,10 +13,33 @@
 
 app.config['CORS_HEADERS'] = 'Content-Type'
 
-resources={r"/job-details": {"origins": "http://localhost:5000"}}
+resources = {r"/job-details": {"origins": "http://localhost:5000"}}
 CORS(app)
 
 
+@app.route("/")
+def index():
+    return render_template("upload.html")
+
+
```

Comment:
let's move this to the addon side, because we might in the future want to make migrate it to a proper framework and have unified styling